content/modules/r/pages/subset.adoc

```diff
@@ -0,0 +1,165 @@
+= `subset`
+
+`subset`  is a function that helps you take subsets of data. By default, `subset` removes NA rows, so use with care. `subset` does not perform any operation that can't be accomplished by indexing, but can sometimes be easier to read.
```

Comment:
`subset` is a function that takes a part of the data. By default, the function removes all `NA` rows.

Note: `subset` doesn't perform any operation that can't be accomplished by indexing.